package.json

```diff
@@ -12,6 +12,7 @@
   "dependencies": {
     "bootstrap": "^4.1.3",
     "connected-react-router": "^6.0.0",
+    "cryptocompare": "^0.7.0",
```

Comment:
we should use etherscan instead which seems to be what status is currently using right now.